ibm/resource_ibm_is_instance.go

```diff
@@ -1680,6 +1704,57 @@ func instanceUpdate(d *schema.ResourceData, meta interface{}) error {
 		return err
 	}
 	id := d.Id()
+
+	if d.HasChange(isInstanceAction) && !d.IsNewResource() {
+
+		_, actiontypeIntf := d.GetChange(isInstanceAction)
```

Comment:
why is get change used? you could have used get if you are not using old value..